src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  men = century ? men.filter(person => {
+    return Math.ceil(person.died / 100) === century;
+  }) : men;
+
+  const callback = (sum, addition) => {
+    return sum + (addition.died - addition.born);
+  };
```

Comment:
```suggestion
  const callback = (sum, addition) => (
    sum + (addition.died - addition.born);
  );
```
Якщо функція тільки робить що відразу щось повертає, то можна замість фігурних дужок писати круглі. Тоді js думає що в тебе функція в 1 строку і не треба писати return